contrib/net/http/roundtripper.go

```diff
@@ -49,9 +49,6 @@ func (rt *roundTripper) RoundTrip(req *http.Request) (res *http.Response, err er
 	if port, err := strconv.Atoi(url.Port()); err == nil {
 		opts = append(opts, tracer.Tag(ext.NetworkDestinationPort, port))
 	}
-	if len(rt.cfg.spanOpts) > 0 {
-		opts = append(opts, rt.cfg.spanOpts...)
```

Comment:
This one was ok. Doesn't modify `spanOpts`.

Reply:
Discussed offline - we'll keep it as is because it was safe before, and this could change order which could have side effects. It's not consistent, but better to not possibly break people.